internal/annotators/base.go

```diff
@@ -16,6 +16,8 @@ package annotators
 import (
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
```

Comment:
Not sure why this got moved. Is it because of an IDE rule? Note that all imports were alphabetized previously which seems fine to me, so what motivated this change?